lib/forms/inputs/MultiSelectInput/MultiSelectInput.css

```
@@ -0,0 +1,75 @@
+@import "@/styles/layers.css";
+@reference "@/styles/tokens.css";
```

Stylelint (17.12.0)

[error] Unexpected unknown at-rule "`@reference`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)